treeherder/intermittents_commenter/commenter.py

```diff
@@ -413,22 +441,30 @@ def build_bug_map(self, bugs, option_collection_map):
         }
         """
         bug_map = {}
-        bug_ids = [b["bug__bugzilla_id"] for b in bugs]
-        bug_summaries = Bugscache.objects.filter(bugzilla_id__in=bug_ids).values(
-            "summary", "bugzilla_id"
-        )
         all_variants = set()
-        for bug, bug_id in zip(bugs, bug_ids):
-            manifest = self.get_test_manifest(bug_summaries.filter(bugzilla_id=bug_id))
+        for bug in bugs:
+            bug_id = bug["bug__bugzilla_id"]
+            all_tests = self.get_tests_from_manifests()
+            test_name = self.get_test_name(all_tests, bug["bug__summary"])
+            manifest = None
             bug_testrun_matrix = []
-            if manifest:
-                testrun_matrix = (
-                    fetch.fetch_testrun_matrix()
-                    if self.testrun_matrix is None
-                    else self.testrun_matrix
-                )
-                self.testrun_matrix = testrun_matrix
-                bug_testrun_matrix = testrun_matrix[manifest]
+            run_count = 0
+            if test_name:
+                manifest = all_tests[test_name][0]
+                if manifest:
+                    tasks_count = self.get_task_labels_and_count(manifest, start_day, end_day)
+                    job_name = bug["job__signature__job_type_name"]
+                    for task_name, count in tasks_count.items():
+                        if task_name in [job_name, job_name.rsplit("-", 1)[0]]:
+                            run_count = count
```

Comment:
an edge case here is:
task_name=test-linux1804-64/debug-mochitest-plain-swr
job_name=test-linux1804-64/debug-mochitest-plain-swr-nofis

so task_name in job_name -> True;  this will end up being misleading.  Maybe to solve this, see if `task_name==job_name or task_name==job_name.rsplit(...)`